server/core/src/lib.rs

```diff
@@ -3,3 +3,6 @@ pub mod database;
 pub mod dto;
 pub mod email;
 pub mod logic;
+
+pub use entity::Id;
+pub use sea_orm::DatabaseConnection;
```

Comment:
Nice touch 🔥

Reply:
I will also make sure to separate our migration crate from the rest that does not necessarily need it (it kinda is a flawed design choice on my part that they do) but that's a bit more work, so lets concentrate on the client side for now.